src/test/java/com/wikia/webdriver/common/core/elemnt/Wait.java

```diff
@@ -24,6 +24,9 @@
    */
 
   private static final int DEFAULT_TIMEOUT = 30;
+  private static final String INIT_MESSAGE = "INIT ELEMENT";
+  private static final String INIT_ERROR_MESSAGE = "PROBLEM WITH ELEMENT INIT";
```

Comment:
The moment when element is initialized by PageFactory. I've used getTagName to generate a findElement event, so I can then get element By and push it to Selectors stack.